src/App.vue

```diff
@@ -8,18 +8,41 @@
 import Component from 'vue-class-component'
 import Vue from 'vue'
 import { Action } from 'vuex-class'
-import { RoomAction, Game, Locale } from '@/store/modules/room'
+import { RoomAction, Game, Locale, GameName, Ship, GameInfo } from '@/store/modules/room'
+import { WowsShipsApiResponse, WowsShipInfoApiResponse } from '@/types/Games/Wows'
+import axios from 'axios'
 
 @Component({
   name: 'TheApp'
 })
 export default class TheApp extends Vue {
-    @Action(`room/${RoomAction.SET_GAME}`) setGame!: (game: Game) => void
-    @Action(`room/${RoomAction.SET_LOCALE}`) setLocale!: (locale: Locale) => void
-    created () {
-      this.setGame(Game['WOT'])
-      this.setLocale(Locale['ENUK'])
+  @Action(`room/${RoomAction.SET_GAME}`) setGame!: (game: Game) => void
+  @Action(`room/${RoomAction.SET_LOCALE}`) setLocale!: (locale: Locale) => void
+  async created () {
+    this.setGame({ name: GameName['WOWS'], ships: [], gameInfo: undefined })
+    this.setLocale(Locale['ENUK'])
+    this.setGame(await this.getWowsApiData())
+  }
+
+  async getWowsApiData (): Promise<Game> {
+    const response: WowsShipsApiResponse = await axios.get('https://api.worldofwarships.eu/wows/encyclopedia/ships/?application_id=d84a218b4fec37003e799f13777bf880')
```

Comment:
extract the application id in a property loaded from an env file

Reply:
We are going to be sticking this in the API soon so I don't want to waste time on making this super pretty considering it is not going to last long